content/en/blog/_posts/2022-05-05-volume-expansion-ga.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: blog
+title: "Volume expansion now available as stable feature in Kubernetes 1.24"
```

Comment:
```suggestion
title: "Kubernetes 1.24: Volume Expansion Now A Stable Feature"
```
The idea is that all the post-release blog articles have titles that start `Kubernetes 1.24: `